schema/resolvers.js

```diff
@@ -16,6 +17,15 @@ const assertMultipleChoiceAnswer = answer => {
   }
 };
 
+const getDefaultAnswerProperties = type => {
+  switch (type) {
+    case "Number":
+      return { required: true, decimals: 0 };
```

Comment:
According to the scenarios, currency should also get a `decimals` property defaulting to zero.

Reply:
I thought we wanted to do that task separately?

Reply:
@adammahmood Yeah my bad, I probably should have made it explicit on the card. The task is to add a `decimals` property and according to Jimi's scenarios that applies to both `Number` and `Currency` answer types.

AFAIK this is the only place that would need to be changed to introduce it for Currency also? As it's such a small change, I think it makes sense to include it in the same PR.

Reply:
Yes and probably best to write some cypress tests to validate on the UI.